### PR TITLE
Tweak headline paragraph as per discussion at #570

### PR DESCRIPTION
**Pull request by robby-dermody**
Please see #570 and [relevant reddit thread](https://www.reddit.com/r/Monero/comments/7tftjy/id_like_the_communitys_feedback_on_my_proposed/). This proposed language seems to have good community support, and I believe it is a definite improvement on the current language.

### CODE REVIEW

**Comment by robby-dermody** [comment]
An alternative is to keep the existing tagline:

> Monero
> Private Digital Currency
> Monero is cash for a connected world. It’s fast, private and secure. With Monero, you are your own bank. You can spend safely, knowing that others cannot see your balances or track your activity.

@dEBRUYNE-1 @erciccione @mattcode55 or other repo maintainers...any feedback?

File: index.html [hunk 1]
```diff
@@ -19,8 +19,8 @@
                     <div class="col-lg-5 col-md-5 col-xs-12">
                         <div class="main-info">
                             <h1 id="main-h1">Monero</h1>
-                            <h2 id="main-h2">Private Digital Currency</h2>
-                            <p id="main-text">Monero is a secure, private, and untraceable cryptocurrency. It is open-source and accessible to all. With Monero, you are your own bank. Only you control and are responsible for your funds. Your accounts and transactions are kept private from prying eyes.</p>
+                            <h2 id="main-h2">True Money</h2>
+                            <p id="main-text">Monero is cash for a connected world. It’s fast, private and secure. With Monero, you are your own bank. You can spend safely, knowing that others cannot see your balances or track your activity.</p>
```
Comment (luigi1111) [inline index.html]: Oxford comma, please. :)

**Comment by robby-dermody** [comment]
@luigi1111 done. Also, reverted the tagline for now until there is more consensus on that.

**Comment by robby-dermody** [comment]
ah fecking heck. attempted a rebase that went haywire. I will close this and reopen a new PR

**Pull request closed without merging**